tools/windows-installer/SshnpdService/Properties/PublishProfiles/FolderProfile.pubxml.user

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121.
+-->
+<Project>
+  <PropertyGroup>
+    <History>True|2024-08-21T14:40:19.4809917Z||;True|2024-08-07T12:44:01.9329536-04:00||;True|2024-08-02T09:49:24.6953198-04:00||;True|2024-07-30T13:48:20.0766899-04:00||;False|2024-07-30T13:47:48.6520841-04:00||;False|2024-07-30T13:46:51.6338566-04:00||;False|2024-07-30T13:45:41.6037496-04:00||;False|2024-07-30T13:45:06.4574331-04:00||;True|2024-07-29T14:30:54.6014075-04:00||;True|2024-07-26T15:55:46.5701039-04:00||;True|2024-07-26T15:55:18.9211215-04:00||;True|2024-07-19T11:54:46.6638001-04:00||;True|2024-07-19T09:56:58.7676027-04:00||;True|2024-07-19T09:45:59.0839738-04:00||;True|2024-07-19T09:45:44.9391570-04:00||;True|2024-07-19T09:45:15.6295347-04:00||;True|2024-07-19T09:44:53.5987691-04:00||;True|2024-07-19T09:44:43.0197788-04:00||;True|2024-07-19T09:38:38.6530424-04:00||;True|2024-07-16T09:33:58.2810892-04:00||;True|2024-07-16T09:30:02.6686572-04:00||;True|2024-07-15T15:08:49.4516190-04:00||;True|2024-07-15T14:50:27.0512831-04:00||;True|2024-07-15T14:46:09.7622736-04:00||;True|2024-07-15T10:46:03.8522878-04:00||;True|2024-07-15T10:44:14.0120102-04:00||;True|2024-07-15T10:36:19.8709496-04:00||;True|2024-07-15T10:35:50.1838770-04:00||;True|2024-07-15T10:35:24.0095129-04:00||;True|2024-07-15T10:34:10.5379066-04:00||;True|2024-07-15T10:33:30.0626619-04:00||;True|2024-07-15T10:32:25.9836209-04:00||;True|2024-07-15T10:22:43.5004201-04:00||;True|2024-07-15T09:57:14.8760545-04:00||;True|2024-07-12T09:11:11.7089159-04:00||;True|2024-07-11T09:48:51.6520694-04:00||;True|2024-07-10T14:09:27.2054713-04:00||;True|2024-07-08T14:26:25.3664065-04:00||;</History>

```

Comment:
What is this for? Looks like this is supposed to be gitignored?

Reply:
resolved in [c79a957](https://github.com/atsign-foundation/noports/pull/1217/commits/c79a957b67e97342cfca7bc8a4d3710adc7771b8)